contracts/contracts/IWarpMessenger.sol

```diff
@@ -23,8 +23,11 @@ interface WarpMessenger {
 
     // sendWarpMessage emits a request for the subnet to send a warp message from [msg.sender]
     // with the specified parameters.
-    // This emits a SendWarpMessage log, which will be picked up by validators to queue the signing of
-    // the corresponding Warp Message after the block containing this log has been accepted.
+    // This emits a SendWarpMessage log from the precompile. When the corresponding block is accepted
+    // the Accept hook of the Warp precompile is invoked with any logs coming from the Warp
+    // precompile address (assuming they were not reverted).
```

Comment:
```suggestion
    // precompile address.
```

Since this refers to accepted logs I don't think we need to exclude reverted.

Reply:
Sounds good, I'll take another look and remove.